### ISSUE TITLE
Kivy usage with 10" touchscreen on Raspberry Pi

### ISSUE THREAD

**Issue by openedhardware** [kivy/kivy#4384]
Dear developers.

I have been developing several projects with Raspberry Pi connected with Official 7" Raspberry Pi touch screen.
Now, I need to expand my screen to 10" touch screen.
 
I have investigated about 10" touchscreen for several days and found one which seems like possible to use Kivy.
http://www.waveshare.com/wiki/10inch_HDMI_LCD_(with_case)
 
I wonder if you could confirm that this is capable with Kivy on Raspberry Pi.

**Comment by dgoadby** [comment]
I am using an 8 inch touchscreen that seems to use a similar controller.  The touchscreen on mine is the eGalax which is a common touch screen controller. 

The touchscreen took a while to sort out but it works perfectly now. You will probably need to use the min/max_abs_x/y parameters in the [input] section of.kivy/config.ini and there is a bug you need to patch before kivy will use them. The eGalax reports absolute co-ordinates which is why you need them.

A very useful utility to install is evtest which is a command line program which will stream the touch screen values continuously and, by touching the corners etc, will indicate the values that your screen is generating.  The evtest output is then used to set the values in 

You may need to update the tslib for eGalax as earlier code didn't recognise it.

The screen was also a PITA and I had to try a number of settings before I got the settings that matched the screen (800x600 in my case).  The various utilities that report the screen capabilities didn't help much as the screen didn't report correctly. In the end it was the HDMI settings in Rpi config.txt that nailed it for me.  After trying many combinations, I eventually set the hdmi_group and hdmi_mode specifically to the mode I wanted and voila! 

I am using Jessie with Kivy 1.9.1. 

In conclusion, I'm sure the screen you are considering should work with Rpi.

**Comment by hunternet93** [comment]
GitHub issues are meant to be used for bugs or feature requests, not general Q&A usage. Please use Kivy's [Google group](https://groups.google.com/forum/#!forum/kivy-users) or [IRC channel](http://webchat.freenode.net/?nick=kvuser.&channels=kivy&uio=d4).

That said, Kivy can use any compatible screen connected to the Pi.

**Comment by matham** [comment]
As mentioned by @hunternet93, this is not the right place for such questions.
